www/src/input/tutorial.md

```diff
@@ -0,0 +1,1790 @@
+## [Installation](#installation) {#installation}
```

Comment:
In the future it would be nice to not have to type the links, but let's leave it for now :)

Reply:
Easy enough to update I think, I'll have a look.